eth-riscv-syscalls/src/lib.rs

```diff
@@ -50,11 +50,13 @@ macro_rules! syscalls {
 // t0: 2, opcode for sstore, a0: storage key, a1: storage value, returns nothing
 // t0: 3, opcode for call, args: TODO
 // t0: 4, opcode for revert, doesn't return
+// t0: 6, opcode for log, a0: data_ptr, a1: data_size, a2: topics_ptr, a3: topics_size, returns nothing
 syscalls!(
     (0, Return, "return"),
     (1, SLoad, "sload"),
     (2, SStore, "sstore"),
     (3, Call, "call"),
     (4, Revert, "revert"),
     (5, Caller, "caller"),
+    (6, Log, "log"),
```

Comment:
Maybe we should use EVM opcodes for the ids here to ensure some more clarity and guidelines. For example this would already conflict with https://github.com/r55-eth/r55/pull/5 just because both were written at the same time.

Reply:
How could I solve it?

Reply:
Just replace the 6 by the EVM log opcode, like done here for keccak https://github.com/r55-eth/r55/pull/5/files#diff-f0fc1b49103455409c2175d93a2ba8f421772819ef6ce75d47665d35de93d624R62